nubis/builder/project.json

```diff
@@ -6,7 +6,7 @@
     ],
     "project_description": "Skeleton nubis module for testing nubis-builder",
     "project_name": "nubis-skel",
-    "project_version": "v1.0.2-dev_{{env `GIT_COMMIT_SHA`}}",
```

Comment:
It's somewhat important to leave the _{{env `GIT_COMMIT_SHA`}} to avoid build names collisions, especially for
things that we expect lots of folks to attempt building